pkg/apis/core/v1alpha1/servicemapping_types.go

```diff
@@ -54,6 +54,11 @@ type ResourceConfig struct {
 	// If unset, the default API version of the service mapping will be used.
 	Version *string `json:"version"`
 
+	// Direct tells if the ResourceConfig is for ConfigConnector directly managed resources.
+	// Directly managed resource does not use Terraform or DCL controller, and do not rely on any TF specified fields like `SkipImport`
+	// A direct ResourceConfig is used to generate g3doc.
```

Comment:
Nit: I think it's not the g3doc, but the reference doc (docs under `Reference` tab in the GCP public site).

Reply:
I was trying to refer to the generate-google3-docs. 
+1 to reference doc. Updated.